app/plugins/adp/src/components/DeliveryProgrammeAdmin/DeliveryProgrammeAdminViewPage.test.tsx

```diff
@@ -0,0 +1,109 @@
+import React from 'react';
+import { errorApiRef } from '@backstage/core-plugin-api';
+import {
+  TestApiProvider,
+  renderInTestApp,
+} from '@backstage/test-utils';
+import { DeliveryProgrammeAdminApi, deliveryProgrammeAdminApiRef } from './api';
+import { DeliveryProgrammeAdminViewPage } from './DeliveryProgrammeAdminViewPage';
+import { waitFor } from '@testing-library/react';
+import { DeliveryProgrammeAdmin } from '@internal/plugin-adp-common';
+import { faker } from '@faker-js/faker';
+import { EntityProvider } from '@backstage/plugin-catalog-react';
+import { Entity } from '@backstage/catalog-model';
+
+const mockDeliveryProgrameAdminApi: jest.Mocked<DeliveryProgrammeAdminApi> = {
```

Comment:
Having the mock created outside of the tests means the same instance persists between tests, which leads to a risk that the tests might be dependant on the order they are executed due to shared state. Could this setup be done inside a setup method which is called at the start of each test to make sure theres no shared state because a new instance is created each time?